lib/cortex/snippets/client.rb

```diff
@@ -1,4 +1,4 @@
-require 'cortex/snippets/client/helper'
+require 'cortex/snippets/client/webpage'
 require 'cortex/snippets/client/railtie' if defined?(Rails)
 require 'cortex-client'
 require 'connection_pool'
```

Comment:
We can nix these two `require`s - `cortex-client` and `connection_pool`. That also means that we can remove `connection_pool` from the `gemspec` (although `cortex-client` would stay, so that `snippets-client` can enforce the correct version of `cortex-client` with the interfaces it expects.